### DISCUSSION TITLE
Design Documents

### DISCUSSION THREAD

**Discussion by dianecawley** [Acumatica/CMMS-Lite#27] [General]
This discussion section holds the design docs for the CMMS lite project.
[Computerized Maintenance Management Software (CMMS).docx](https://github.com/Acumatica/CMMS-Lite/files/9890929/Computerized.Maintenance.Management.Software.CMMS.docx)
[CMMS Lite Functional Specification.docx](https://github.com/Acumatica/CMMS-Lite/files/9890930/CMMS.Lite.Functional.Specification.docx)

**Comment by MvF-Evangelista** [comment]
Diane, please add these documents to the Main README file.
